tools/screenshots/generate-integration-docs-screenshot

```diff
@@ -61,7 +61,7 @@ def create_integration_bot(integration: Integration, bot_name: str | None = None
     owner = get_user_by_delivery_email("iago@zulip.com", realm)
     bot_email = f"{integration.name}-bot@example.com"
     if bot_name is None:
-        bot_name = f"{integration.name.capitalize()} Bot"
+        bot_name = f"{integration.display_name} Bot"
```

Comment:
It's probably worth having a #design conversation about whether we want to have "Bot" in these names, or should be encouraging just "GitHub" rather than "GitHub Bot". (I have no specific reason to believe we should change that, just noting it's a thing we've not discussed seriously in 5+ years).